packages/flutter_tools/test/general.shard/web/devfs_web_ddc_modules_test.dart

```diff
@@ -116,1386 +61,1212 @@ void main() {
   setUp(() {
     httpServer = FakeHttpServer();
     linux = FakePlatform(environment: <String, String>{});
-    windows = FakePlatform(
-        operatingSystem: 'windows', environment: <String, String>{});
+    windows = FakePlatform(operatingSystem: 'windows', environment: <String, String>{});
     logger = BufferLogger.test();
     testbed = Testbed(
-        setup: () {
-          webAssetServer = WebAssetServer(
-            httpServer,
-            packages,
-            InternetAddress.loopbackIPv4,
-            <String, String>{},
-            <String, String>{},
-            NullSafetyMode.unsound,
-            usesDdcModuleSystem,
-            webRenderer: WebRendererMode.canvaskit,
-            useLocalCanvasKit: false,
-          );
-          releaseAssetServer = ReleaseAssetServer(
-            globals.fs.file('main.dart').uri,
-            fileSystem: globals.fs,
-            flutterRoot: null, // ignore: avoid_redundant_argument_values
-            platform: FakePlatform(),
-            webBuildDirectory: null, // ignore: avoid_redundant_argument_values
-            needsCoopCoep: false,
-          );
+      setup: () {
+        webAssetServer = WebAssetServer(
+          httpServer,
+          packages,
+          InternetAddress.loopbackIPv4,
+          <String, String>{},
+          <String, String>{},
+          NullSafetyMode.unsound,
+          usesDdcModuleSystem,
+          webRenderer: WebRendererMode.canvaskit,
+          useLocalCanvasKit: false,
+        );
+        releaseAssetServer = ReleaseAssetServer(
+          globals.fs.file('main.dart').uri,
+          fileSystem: globals.fs,
+          flutterRoot: null,
+          platform: FakePlatform(),
+          webBuildDirectory: null,
+          needsCoopCoep: false,
+        );
+      },
+      overrides: <Type, Generator>{Logger: () => logger},
+    );
+  });
+
+  @isTest
+  void runTestbed(
```

Comment:
I think `runInTestbed` would be a more descriptive name.

Reply:
I don't have any familiarity with how `testbed.run()` works; thanks for helping me improve this!